funnel/views/project.py

```diff
@@ -276,37 +277,32 @@ class ProjectView(
     ProjectViewMixin, DraftViewMixin, UrlChangeCheck, UrlForView, ModelView
 ):
     @route('')
-    @render_with('project.html.jinja2')
+    @render_with(html_in_json('project.html.jinja2'))
     @requires_roles({'reader'})
     def view(self):
-        transition_form = ProjectTransitionForm(obj=self.obj)
-        rsvp_form = RsvpTransitionForm()
-        current_rsvp = self.obj.rsvp_for(current_auth.user)
-        featured_proposals = self.obj.proposals.filter_by(featured=True)
         return {
             'project': self.obj.current_access(),
-            'current_rsvp': current_rsvp,
-            'rsvp_form': rsvp_form,
-            'transition_form': transition_form,
-            'featured_proposals': featured_proposals,
+            'featured_proposals': [
+                _p.current_access()
```

Comment:
Needs `datasets=('without_parent', 'related')` (assuming Proposal model defines a 'without_parent' dataset, which it should).